chalicelib/check_setup.json

```diff
@@ -201,6 +201,18 @@
             }
         }
     },
+    "check_bio_feature_organism_name": {
+        "title": "BioFeatures with missing or mismatched check_bio_feature_organism_name",
```

Comment:
I'm guessing this was a mistaken autocomplete and it ought to be more like "Biofeatures with missing or mismatched organism name"?

Reply:
Yep - thanks!